src/org/zaproxy/zap/extension/httpsinfo/resources/Messages.properties

```diff
@@ -0,0 +1,38 @@
+# A ZAP extension which adds a right click menu item to all of the main
```

Comment:
Is this description really needed? (Good to have in the JavaDoc of the extension.)